eng/testing/runsettings.targets

```diff
@@ -21,7 +23,7 @@
       <RunSettingsFileContent>$([System.IO.File]::ReadAllText('$(RunSettingsInputFilePath)'))</RunSettingsFileContent>
       <RunSettingsFileContent>$(RunSettingsFileContent.Replace('$$RESULTSDIRECTORY$$', '$(ArtifactsTestResultsDir)')
                                                       .Replace('$$SOLUTIONDIRECTORY$$', '$(RepoRoot)')
-                                                      .Replace('$$TARGETPLATFORM$$', '$(TargetPlatform)')
+                                                      .Replace('$$TARGETPLATFORM$$', 'x64')
```

Comment:
What's the purpose of hardcoding x64? The change in line 4 indicates that `TargetPlatform` acquires that same value by default when it is empty.

Reply:
that was a bug, fixed.